components/admin_console/workspace-optimization/dashboard.tsx

```diff
@@ -244,26 +244,70 @@ const WorkspaceOptimizationDashboard = (props: Props) => {
     }, []);
 
     const accData: AccordionItemType[] = Object.entries(data).map(([accordionKey, accordionData]) => {
-        const items = accordionData.items.map((item) => (
-            <AccordionItem
-                key={`${accordionKey}-item_${item.id}`}
-                iconColor={getItemColor(item.status)}
-            >
-                <h5><i className={classNames('icon', {'icon-check-circle-outline': item.status === 'none', 'icon-alert-outline': item.status === 'warning', 'icon-alert-circle-outline': item.status === 'error'})}/>{item.title}</h5>
-                <p>{item.description}</p>
-            </AccordionItem>
-        ));
+        const chipsInfo: { [key: string]: number } = {
+            info: 0,
+            warning: 0,
+            error: 0,
+        };
+        type ChipsInfoKey = keyof typeof chipsInfo;
+        const items: React.ReactNode[] = [];
+        accordionData.items.forEach((item) => {
+            items.push((
+                <AccordionItem
+                    key={`${accordionKey}-item_${item.id}`}
+                    iconColor={getItemColor(item.status)}
+                >
+                    <h5><i className={classNames('icon', {'icon-check-circle-outline': item.status === 'none', 'icon-alert-outline': item.status === 'warning', 'icon-alert-circle-outline': item.status === 'error'})}/>{item.title}</h5>
+                    <p>{item.description}</p>
+                </AccordionItem>
+            ));
+            if (chipsInfo[item.status as ChipsInfoKey] !== undefined) {
+                chipsInfo[item.status as ChipsInfoKey] += 1;
+            }
+        });
+        const chipsList = Object.entries(chipsInfo).map(([chipKey, count]) => {
+            if (count === 0) {
+                return false;
+            }
+            let id;
+            let defaultMessage;
+
+            switch (chipKey) {
+            case 'info':
+                id = 'admin.reporting.workspace_optimization.suggestions';
+                defaultMessage = 'Suggestions';
+                break;
+            case 'warning':
+                id = 'admin.reporting.workspace_optimization.warnings';
+                defaultMessage = 'Warnings';
+                break;
+            case 'error':
+                id = 'admin.reporting.workspace_optimization.problems';
+                defaultMessage = 'Problems';
+                break;
+            }
+
+            return (
+                <Chip
+                    key={chipKey}
+                    id={id}
+                    defaultMessage={`${defaultMessage}: ${count}`}
+                    className={chipKey}
+                />
+            );
+        });
```

Comment:
we could filter out the `false` entries from the array here:

```suggestion
        }).filter(Boolean);
```

Reply:
nicer but implies another internal loop over the results. keeping current implementation.